admin/tabs/activity/clienterrors/ClientErrorsModel.ts

```diff
@@ -113,7 +113,7 @@ export class ClientErrorsModel extends HoistModel {
 
         this.formModel = new FormModel({
             readonly: true,
-            fields: this.gridModel.columns.map(it => ({name: it.field, displayName: it.headerName}))
+            fields: this.gridModel.getLeafColumns().map(it => ({name: it.field, displayName: it.headerName as string}))
```

Comment:
Wonder if we should expose `leafColumns` as a getter on the GridModel, given that we are going to be (correctly) using it more going forward